### PR TITLE
[CBRD-24947] Add test case for OUT parameters must be initialized to NULL on the PL/CSQL

### PR DESCRIPTION
**Pull request by swi0110**
Refer to http://jira.cubrid.org/browse/CBRD-24947

### CODE REVIEW

File: sql/_05_plcsql/_01_testspec/_05_bug_fix/cases/21_out_param_init_null.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
+--+ server-message on
+-- Verified for CBRD-24947
+
+create or replace procedure test_out_param(i OUT int) as
+    procedure print_null_or_not(k int) as
+    begin
+        dbms_output.put_line(case when k is null then 'null' else 'non-null' end);
+    end;    procedure inner(j OUT int) as
```
Comment (kwonhoil) [inline sql/_05_plcsql/_01_testspec/_05_bug_fix/cases/21_out_param_init_null.sql]: 아래와 같이 수정되는게 좋겠습니다.

end;

procedure inner(j OUT int) as

Comment (swi0110) [reply sql/_05_plcsql/_01_testspec/_05_bug_fix/cases/21_out_param_init_null.sql]: 반영했습니다
리뷰 감사합니다

**Review by hyunikn — APPROVED** [review]

**Review by hiclass — APPROVED** [review]

**Review by ssihil — APPROVED** [review]

**Review by kwonhoil — APPROVED** [review]
